scan_rr.go

```diff
@@ -846,6 +846,70 @@ func (rr *CSYNC) parse(c *zlexer, o string) *ParseError {
 	return nil
 }
 
+func (rr *ZONEMD) parse(c *zlexer, o string) *ParseError {
+	l, _ := c.Next()
+	if (l.token == "\\#") {
```

Comment:
This is definitely not the correct place to do this nor the correct pull request to do this in, please back this out.

Reply:
where to, then?

Reply:
It looks like we only support RFC3597 in `(*ZoneParser).Next` if we don't recognise the RR type, that does seem like a bug to me. That bug needs to be fixed in the ZoneParser to work for *all* record types using the existing wire format parsing functions not here in an ad-hoc way for this specific one.

Reply:
I've opened #1211 to add support for this.